src/main/java/com/crosswordservice/controller/CrosswordController.java

```diff
@@ -51,7 +52,8 @@ public List<Question> inputTestData(){
      */
     @PostMapping("/configurations")
     public Configuration saveConfiguration(@RequestBody Configuration configuration){
-        configurationRepository.save(configuration);
+        Configuration persistentConfiguration = new Configuration(configuration.getName());
```

Comment:
Is it right that only the name of the configuration is saved. Isn't there a option to post the whole configuration?

Reply:
They also have a List of questions, haven't they?

Reply:
I think at creation of a configuration it would be useful to be able to directly add a set of questions